### PR TITLE
Multiple small design changes

### PR DESCRIPTION
**Pull request by allinox**
## Description

As requested by UX, these changes have been made:
- Heading of Api-delegation has been swiched back to dark mode
- The borders of the actionBar has been set to use default design system values
- Changed remove button on search-actionBars on single rights to say undo and use the undo symbol instead
- Added minus icon on remove button in the bars inside the CollectionBar

In addition to this, the enums in Page have been removed, using instead a string type with a finite set of options

## Related Issue(s)
- #468

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/features/apiDelegation/received/OverviewPage/OverviewPage.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { useTranslation } from 'react-i18next';
 import * as React from 'react';
 
-import { Page, PageHeader, PageContent, PageSize, PageContainer, PageColor } from '@/components';
+import { Page, PageHeader, PageContent, PageSize, PageContainer } from '@/components';
```
Comment (Albertlarsen) [inline src/features/apiDelegation/received/OverviewPage/OverviewPage.tsx:4]: PageSize is defined but never used

File: src/features/singleRight/delegate/ChooseServicePage/ResourceActionBar/ResourceActionBar.tsx [hunk 2]
```diff
@@ -2,7 +2,12 @@
 import * as React from 'react';
 import { useMemo, useState } from 'react';
 import { Button, Paragraph } from '@digdir/design-system-react';
-import { PlusCircleIcon, MinusCircleIcon, ExclamationmarkTriangleIcon } from '@navikt/aksel-icons';
+import {
+  PlusCircleIcon,
+  MinusCircleIcon,
```
Comment (Albertlarsen) [inline src/features/singleRight/delegate/ChooseServicePage/ResourceActionBar/ResourceActionBar.tsx:7]: MinusCircleIcon is defined but never used

File: src/localizations/en.json [hunk 3]
```diff
@@ -17,6 +17,7 @@
     "search": "Search",
     "add": "Add",
     "remove": "Remove",
+    "undo": "Undo",
```
Comment (Albertlarsen) [inline src/localizations/en.json:20]: Should we remove the api_delegation.undo? I can see arguments for both keeping it as it is and for removing it

**Review by Albertlarsen — APPROVED** [review]
N🧊 work 🤩

File: src/localizations/en.json (hunk 3 above)
Comment (allinox) [reply src/localizations/en.json:20]: Well, they're the same and undo is a such a common thing that it should probably be in the common section anyway. So I'll just remove it from api-delegation

**Pull request merged**